autoload/test/scala/sbttest.vim

```diff
@@ -0,0 +1,39 @@
+if !exists('g:test#scala#sbttest#file_pattern')
+  let g:test#scala#sbttest#file_pattern = '\v^.*.scala$'
```

Comment:
Is it possible to make the test file regex match test files but not application files, e.g. `\vsrc/test/scala/.*\.scala$`? That helps make test.vim smarter, e.g. if you run tests on an application file, it would detect that it's not a test file and run tests for the last test file instead.

Reply:
Good point, but I looked at our codebase at work, and it looks like it is overwhelmingly common for people to include "Test", "Suite", or both in the filename. I will work on an improved regex; so far, I created a fixture for scala, will create a spec and extend README tomorrow.